packages/pinia/src/store.ts

```diff
@@ -452,12 +452,16 @@ function createSetupStore<
   const store: Store<Id, S, G, A> = reactive(
     assign(
       __DEV__ && IS_CLIENT
-        ? // devtools custom properties
+        ?
```

Comment:
let's keep the whole thing in one ternary only. It's fine if `_hmrPayload` is there too

Reply:
Updated. Thanks.